hosting/k8s/helm/templates/webapp.yaml

```
@@ -38,6 +75,55 @@ spec:
+          image: bitnami/kubectl:1.28
+          securityContext:
+            runAsUser: 1000
+            runAsNonRoot: true
+          command:
+            - /bin/bash
+            - -c
+            - |
+              TOKEN_FILE="/home/node/shared/worker_token"
+              SECRET_NAME="{{ include "trigger-v4.fullname" . }}-worker-token"
+              NAMESPACE="{{ .Release.Namespace }}"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)